### ISSUE TITLE
Dead or Alive 3 - Crash on boot

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#69]
_From @ergo720 on July 29, 2017 21:10_

PAL, win10, d44c3930

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1185245/KrnlDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1185557/Xbe.txt)

![doa3](https://user-images.githubusercontent.com/30000751/28748243-4df78684-74a2-11e7-873c-934783aa06cd.PNG)

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#550_

**Comment by ergo720** [comment]
DOA3 gets in game now and I could win a fight. There's a problem with the text in the menus which is not rendered at all though. Sound is good as well. Tested with HLE, https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/commit/587fc9da2d8a741f2cab9cccf52ce6807e88f9c7
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1880994/KrnlDebug.txt)

**Comment by Shideravan** [comment]
d784ce1f (2018-04-07): tested using HLE mode, Windows 10, Nvidia GPU

![image](https://user-images.githubusercontent.com/12163923/38456476-670d311c-3a5b-11e8-8e05-80ab6831aa74.png)

**Comment by ObiKKa** [comment]
I think this video is worth showing because of the almost slick graphics and beautiful performance. Though, do try to 'patch up that suit' :blush: :smiling_imp:, Patrick! Hehe. Still, lucky JGG gets to record it for posterity!

Also, please be sure to note the quick flash of a swizzled texture/screen during the load window stage ("Get Ready" after an intro cutscene) and after a round ends (Only if you lose. Weird.). The GUI and characters' ground shading (Like in the screens above) are incomplete.
I like the error in the second stage which is funny - both fighters' reflections are of their opposite on the shiny ground! :satisfied:
Near the end after a boss battle a flickering FMV is briefly shown.

- [Cxbx-Reloaded Xbox Emulator - Dead or Alive 3 Ingame! HW Mode+FXAA (d784ce1)](https://www.youtube.com/watch?v=REUjQ-acX80) (John GodGames Emus. Published on Apr 7, 2018.)

**Comment by Shideravan** [comment]
I witnessed several graphical glitches in this game in my test in the lastest CXBX version (6b6f6265)
Native resolution. Widescreen mode. 
HLE emulation

[CXBX-Reloaded (6b6f6265) | Dead or Alive 3](https://youtu.be/yCQYWiytS40)

My Specs:
Intel Core i7-4790 @ 3.60GHz
Galax NVIDIA GeForce 1070 EXOC
Gigabyte B85M-D3PH
HyperX Fury 1600 16GB (2x8)
Kingston SSD 120GB
Corsair CX-750 (750W)
Windows 10 Pro

**Comment by gandalfthewhite19890404** [comment]
DoA 3 regressed? On latest master I have blackscreen.
Need confirm from users.

**Comment by Juancrack97** [comment]
I downloaded the lastest version of the emulator (c86688bf) and the game doesn't working for me, runs very well but after the intro crashes.
I will show you the specifications of my PC
CPU: intel core i5-4300M @2.60Ghz
GPU: intel HD graphics 4600
RAM: 8gb
OS: Windows 7 x64 professional
Any solution?

**Comment by haradons** [comment]
It works for 46c079cf
![image](https://user-images.githubusercontent.com/13687549/40273336-5e83a8a8-5b73-11e8-8fbb-60d1ea6f484d.png)

**Comment by Juancrack97** [comment]
![test](https://user-images.githubusercontent.com/25289102/40274702-1c4bc472-5ba3-11e8-9148-b611fc26096a.png)
Now I updated the emulator again, but the problem persists, the intro it's OK but after this crashes :(

**Comment by SonicThePS4** [comment]
The Cxbx Reloaded latest update has been going very well but I think Dead Or Alive 3 lost the Main Menu Texts again. :( But I have to say, Sega GT Online looks Fantastic right now. You guys are amazing. :D

![image](https://user-images.githubusercontent.com/39503785/46251401-acfb9800-c41f-11e8-8e39-49e8bc81b999.png)

**Comment by sonicjam717** [comment]
1a584133 Oct. 18 2018

Game is having even more video issue for example video scrambling. But can be fixed with a speed hack checking Host Backbuffer. It reverts to the same issue like the picture sonictheps4 shared.

**Comment by SonicThePS4** [comment]
@sonicjam717 really? thanks i'll try it out.

**Comment by MattPonton** [comment]
From the brief time I've spent with the game,

Latest Stable Release (July 13, 2018) has the game playable. Noticeable issues are as follows:

- Any stage with fire or wind comes in very loud and corrupted.
- Character rendering has tris that don't render completely.
- Most subtitles and text is not rendering, though its shadow layer renders.

Latest Dev Release (Nov 25, 2018) has the game unplayable.

- Any in-game view is a skewed interlaced screen making it impossible to visually determine what's going on.
- Character select screen renders at 1/4 the size and 1/4 the frame rate.

**Comment by LukeUsher** [comment]
@MattPonton did you try with the 'Render to Host Backbuffer' speed hack? That is known to improve results in certain titles, this one included.

**Comment by MattPonton** [comment]
> @MattPonton did you try with the 'Render to Host Backbuffer' speed hack? That is known to improve results in certain titles, this one included.

Yes, thank you for reminding me. It's the only setting that differed from the stable build I was running. With that set, the game can be "playable" on latest, though the UI seems regressed (invisible) now. Additionally, the Snow shader appears to be showing corrupted snow which may be related.

Here's some screens of the latest development build running those settings:
![image](https://i.snag.gy/wZ9XDF.jpg)
![image](https://snag.gy/Wx5s8A.jpg)
![image](https://i.snag.gy/ELentI.jpg)

In comparison, the latest stable build running the same settings:
![image](https://snag.gy/I9jVok.jpg)
![image](https://i.snag.gy/MnuYhl.jpg)
![image](https://i.snag.gy/uqoNQF.jpg)

**Comment by ouzzzz** [comment]
Hello guys, i tried to launch this game and it worked. I can play at full speed 60fps with the 3 speed hacks enabled. There is somes maps and outfits who are a little bit bugged and the white subtitles became black. 
![doa 3 working](https://user-images.githubusercontent.com/47872664/53209573-2bf5a500-363b-11e9-82ef-8218fa086344.png)

**Comment by ouzzzz** [comment]
I can upload a video if you want

**Comment by bamncan** [comment]
As an update, tried out the Aug 25, 2019 build today.

**Pros**
- Character models are all rendering correctly it seems. No more vertex holes or reflection issues
- Character shadows are still rendering too high but don't impede gameplay.

**Cons**
- In-game HUD is nearly non-existant, with the only exception being the lifebars.
- You have to navigate menus blind. Possibly the same issue as the invisible HUD. I presume it might be a specific texture render? but purely a guess.
- Subtitles and menu option information/text is also blind.
- Major slowdown (more than the actual game) when you defeat the story boss.
- Wind & Fire sound effects are distorted, sound is cracking.

Most of the HUD issues weren't occuring in the last known build that showed the HUD. I believe it was in a June 18, 2018 build of the emulator (da06670). Subtitles and text were still not working, but the shadows of them were rendering so it made it more playable.

**Comment by bamncan** [comment]
Also, I noticed the Xbe.txt file in the original post is only of the European/PAL version (Recognized as 3.2). The American (3.0) and Japanese (3.1) versions were each different from the PAL version which came last.

Attaching the XBE dumps of all three.
[doa3.0_xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3538657/doa3.0_xbe.txt)
[doa3.1_xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3538658/doa3.1_xbe.txt)
[doa3.2_Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3538659/doa3.2_Xbe.txt)

**Comment by SonicThePS4** [comment]
Dead or Alive 2 & Dead or Alive 3 is running perfect but theirs a new plant/trees shading issues that's going on, as you can see.

![image](https://user-images.githubusercontent.com/39503785/64652674-6ca7e680-d3f2-11e9-89db-35d927b9f806.png)

**Comment by DanteSCR2685** [comment]
Still crashes during intro. The latest build from 26th of December.

**Comment by ObiKKa** [comment]
Many screenshots in several posts above from different users give the matching title ID: [TC-001] [v1.01].

**Comment by bamncan** [comment]
@ObiKKa @Margen67 Just to clarify, there are three different versions of Dead or Alive 3, their title IDs differ between regions. The North American (Released Nov 2001) and the Japanese (Released Feb 2002) versions though different in code and versioning, share the same Title ID of 54430001. However, the European version which released last in Feb 2002, has a different Title ID of 4d53002d.
CXBX-R reports the NTSC pair in the App Title as [TC-001] [v1.01], and the PAL version as [MS-045] [v1.01]

```
Region | Title ID | App Display ID   
NA     | 54430001 | TC-001 v1.01
JP     | 54430001 | TC-001 v1.01
EU     | 4d53002d | MS-045 v1.01
````

**Comment by bartlomiejduda** [comment]
Main menu is still invisible in the latest build (Sep 8 2020)
![obraz](https://user-images.githubusercontent.com/5502283/94070626-d8d56c80-fdf2-11ea-84be-70007822c629.png)
